packages/devui-vue/devui/tooltip/src/use-tooltip.ts

```diff
@@ -3,7 +3,7 @@ import type { Ref } from 'vue';
 import { debounce } from 'lodash';
 import { TooltipProps, BasePlacement, UseTooltipFn } from './tooltip-types';
 
-const TransformOriginMap: Record<string, string> = {
+export const TransformOriginMap: Record<string, string> = {
```

Comment:
此处应该使用小驼峰，一般类或者类型采用大驼峰，普通变量应该使用小驼峰
```suggestion
export const transformOriginMap: Record<string, string> = {
```